src/withFacets.js

```diff
@@ -149,7 +149,8 @@ function withFacets(WrappedComponent) {
       } else if ([FACETS.SUBJECTS_SHARED, FACETS.SUBJECTS_HELD_BY].includes(facetName)) {
         params.query = 'value=*';
       } else if (cqlQuery && Object.values(browseCallNumberOptions).includes(queryIndex)) {
-        params.query = 'callNumber=""';
+        const callNumberType = queryIndex === browseCallNumberOptions.CALL_NUMBERS ? '' : queryIndex;
+        params.query = `callNumberType="${callNumberType}"`;
```

Comment:
Should the `query` also include the selected facet options as implemented in the `Search` lockup?
e.g. `query: callNumberType="local" and shared=="true"`

Reply:
Possibly, but that's out of scope of this ticket. This needs to additionally investigated and estimated

Reply:
I mean only the `Shared` facet. If we only use the `Shared` facet in the `Search` lookup and select `Yes`, the facet request query will contain the selected options, but the `Shared` facet in the `Browse` search doesn't append selected options to facet request.